opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationGraph.jsx

```diff
@@ -166,6 +166,9 @@ const investigationGraphStixCoreObjectQuery = graphql`
         first_seen
         last_seen
       }
+      ... on MalwareAnalysis {
```

Comment:
Maybe, you can try to use the representative mechanism here (a bit challenging I suppose)

I see usage of defaultValue in Graph.buildGraphData method